### PR TITLE
LMS Integration

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: lib/users.js [hunk 1]
```diff
@@ -245,6 +245,36 @@ exports.register = function (server, options, next) {
...
+          responseMessages: [
+            {code: 200, message: 'OK'}
+          ]
+        }
+      }
+    }
+  }, {
+    method: 'POST',
+    path: options.basePath + '/lms/badge',
+    handler: actHandlerCheckCertificate,
```
Comment (DanielBrierton) [inline lib/users.js]: Not really descriptive of what it actually does. Perhaps should be specific, or take an argument for the act executed after checking the cert.

Comment (Wardormeur) [inline lib/users.js]: Not sure to get what you mean, it's returning the link used for the front-end

Comment (Wardormeur) [inline lib/users.js]: explained irl, got it :)

**Comment by DanielBrierton** [comment]
Tested, and all works well. The only issue I found was that because I didn't add any tags to my badge for it, it didn't show on my profile. Perhaps we should have an Other section for those that don't match the tags we look for?

**Pull request closed without merging**